js/product-page.js

```diff
@@ -1,15 +1,25 @@
+// location.search = location.search + "_" + "querystring";
+// let ss = location + "/newURL"
+
 const product = location.search.substr(1).split("_");
 const productCategory = product[0];
 const productId = product[1];
 
 var xmlhttp = new XMLHttpRequest();
-xmlhttp.onreadystatechange = function() {
+xmlhttp.onreadystatechange = function () {
   if (this.readyState == 4 && this.status == 200) {
     var data = JSON.parse(this.responseText);
     if (data[productCategory].length < productId) {
       eror404();
       return;
     }
+    changeUrl(data)
+
+    addPointTooBreadcrumbMap(product[0], "#");
```

Comment:
a little bit different
you should have an array with breadcrumbs like
`breadcrumbs = [ { name: product[0], link: "#" }, { name: data["drinks"][productId].category, link: "#" }, { name: data["drinks"][productId].name, link: "#" }]`
and one function call `addPointTooBreadcrumbMap(breadcrumbs);`
that's it. function should create as much breadcrumbs as you sent in array